content/english/python-tensorflow/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Python: Machine Learning Basics using Tensorflow"
-description: "Learn basic image classification and train ML models with Tensorflow"
+title: "Python: Machine Learning Basics using TensorFlow"
```

Comment:
Make sure these changes don't conflict with @beagandica

Reply:
don't see any merge conflicts for this file, resolving